spec/FrameSpec.js

```diff
@@ -0,0 +1,161 @@
+describe('Frame', function(){
+
+  beforeEach(function(){
+    frame = new Frame(false);
+  });
+
+  it('starts with a total score of 0 for first bowl of frame', function(){
```

Comment:
These two first tests, and most of the other ones do not need to exist. They are testing *state* (instance variables).
As a rule, what you want to test are *behaviours*, which usually mean testing externally accessible methods (public methods) of the class.

Can you guess why that is?